modules/navier_stokes/src/postprocessors/NSEntropyError.C

```diff
@@ -23,8 +23,8 @@ NSEntropyError::validParams()
   params.addClassDescription("Computes entropy error.");
   params.addRequiredParam<Real>("rho_infty", "Freestream density");
   params.addRequiredParam<Real>("p_infty", "Freestream pressure");
-  params.addRequiredCoupledVar(NS::density, "density");
-  params.addRequiredCoupledVar(NS::pressure, "pressure");
+  params.addRequiredCoupledVar("rho", "density");
+  params.addRequiredCoupledVar("pressure", "pressure");
```

Comment:
Why did this have to happen? Why couldn't these names still be available for the compressible Navier Stokes objects to use?

Reply:
I guess this is the change from `NS::pressure` to `pressure`. I forgot the exact reason for changing this, possibly due to pressure naming with either `p` or `pressure`. I think using fixed parameter name is a better practice. Potentially `NS.h` can change.

Reply:
I feel like `NS.h` changing is not any more likely than someone coming along and trying to change the parameter name in `NSEntropyError.C` or other places. A nice thing about using something like `NS::pressure` everywhere is that if you ever did want to change the parameter name, you would only have to do it in one place. This isn't going to be the hill that I die on, but I would like to avoid having to regold things...

Reply:
I think I remembered. I changed `NS.h` here for pressure name because INS is using `p` but here it has `pressure`. I guess NS.h was designed for CNS actions, so it is not an problem before. Now to be consistent, I choose to change the pressure name, which forces us to change lots of input files where `pressure` is used as the parameter name. This is the reason why I changed the parameter name not to use names from `NS` namespace. I think it is unlikely we change the parameter name in the future. Note that even we can change the parameter name in one place, it will affect the input files, which I can see will be a big pain. I double checked that the only regolding in this commit is due to the pressure variable name change.